js/app.js

```diff
@@ -39,8 +39,26 @@ handleMainNav = function() {
   });
 };
 
+handleMobileNav = function() {
+  $('.mobilenav').on('click', '.tab', function(){
+    $('.mobilenav').fadeOut(500);
+    $('.top-menu').removeClass('top-animate');
+    $('body').removeClass('noscroll');
+    $('.mid-menu').removeClass('mid-animate');
+    $('.bottom-menu').removeClass('bottom-animate');
+    var val = $(this).attr('data-content');
+    $('.tab-content').hide();
+    $('.tab-content').each(function(index){
+      if($(this).attr('id') === val){
```

Comment:
You can select ids easilly with jQuery with just $('#idname'), where 'idname' is the name of the id. You can also concatenate strings in JS. You really don't need an .each() loop here.